### PR TITLE
Feedback implementation for the Workshop

### PR DESCRIPTION
**Pull request by MarkusGarmeister**
## fixes KILTProtocol/ticket#3455
I made several changes to improve the project:

Updated the JS code for ES6 compatibility:
 -Added file extensions to the imported files.
 -Changed the require statements to ES6 import syntax.

Added a generateKeypairs file to the attester module.
Included terminal commands for quick creation of folder structures and files. 
Modified the output of the DID to make it less confusing.

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [x] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
@MarkusGarmeister Fixed merge conflicts, I think you started work before I made changes. Will now look at the code side.

File: code_examples/sdk_examples/src/workshop/attester/generateKeypairs.ts [hunk 1]
```diff
@@ -12,16 +12,13 @@ export function generateKeypairs(mnemonic = mnemonicGenerate()): {
 
   const assertionMethod = Kilt.Utils.Crypto.makeKeypairFromUri(mnemonic)
 
-  const capabilityDelegation = Kilt.Utils.Crypto.makeKeypairFromUri(mnemonic)
```
Comment (ChrisChinchilla) [inline code_examples/sdk_examples/src/workshop/attester/generateKeypairs.ts]: @MarkusGarmeister This is needed: https://github.com/KILTprotocol/docs/actions/runs/9696806639/job/26759525902?pr=318

**Comment by ChrisChinchilla** [comment]
Also seems to be a lot of text editor / IDE formatting differences in the PR, markdown doesn't really have standards, so I don't mind allowing these in, but they're not really necessary and someone else's changes will probably change them again. I use `markdownlint` personally.

File: code_examples/sdk_examples/src/workshop/attester/generateKeypairs.ts (hunk 1 above)
Comment (MarkusGarmeister) [reply code_examples/sdk_examples/src/workshop/attester/generateKeypairs.ts]: I added the variable to the file

**Comment by ChrisChinchilla** [comment]
@MarkusGarmeister Something is wrong with our CI tests, I ran them locally and everything worked OK, so will approve this now

**Review by ChrisChinchilla — APPROVED** [review]

**Pull request merged**